service.unmanic/resources/fetch_dependencies.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+    unmanic
+
+    Written by:               Josh.5 <jsunnex@gmail.com>
+    Date:                     08 Dec 2020, (21:07 PM)
+
+    Copyright:
+        Copyright (C) 2021 Josh Sunnex
+
+        This program is free software: you can redistribute it and/or modify it under the terms of the GNU General
+        Public License as published by the Free Software Foundation, version 3.
+
+        This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the
+        implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+        for more details.
+
+        You should have received a copy of the GNU General Public License along with this program.
+        If not, see <https://www.gnu.org/licenses/>.
+
+"""
+
+import os
+import time
+import shutil
+
+import xbmc
+import xbmcvfs
+import xbmcgui
+import xbmcaddon
+from resources import downloader, extract
+
+__addon__ = xbmcaddon.Addon()
+__path__ = __addon__.getAddonInfo('path')
+__addonname__ = __addon__.getAddonInfo('name')
+__version__ = __addon__.getAddonInfo('version')
+__icon__ = __addon__.getAddonInfo('icon')
+__ID__ = __addon__.getAddonInfo('id')
+__language__ = __addon__.getLocalizedString
+__profile__ = xbmcvfs.translatePath(__addon__.getAddonInfo('profile'))
+
+ffmpeg_url_template = "https://johnvansickle.com/ffmpeg/builds/ffmpeg-git-{}-static.tar.xz"
```

Comment:
Not sure if we want to download from non-official mirrors? @enen92

Reply:
These builds are listed on the official FFmpeg website:
https://ffmpeg.org/download.html
"Linux Static Builds"

Reply:
But Kodi already has ffmpeg include, can you use that?